doc/using_pytorch.rst

```diff
@@ -566,11 +570,91 @@ The function should return a byte array of data serialized to content_type.
+Typically, you save a PyTorch model as a file with extension ``.pt`` or ``.pth``.
+To do this, you need to:
+
+* Write an inference script.
+* Package the model artifacts into a tar.gz file.
+* Upload the tar.gz file to an S3 bucket.
+* Create the ``PyTorchModel`` object.
+
+Write an inference script
+-------------------------
+
+You must create an inference script that implements (at least) the ``predict_fn`` function that calls the loaded model to get a prediction.
+Optionally, you can also implement ``input_fn`` and ``output_fn`` to process input and output. 
+For information about how to write an inference script, see `Serve a PyTorch Model <#serve-a-pytorch-model>`_.
+Save the inference script as ``inference.py`` in the same folder where you saved your PyTorch model. 
+
+Package model artifacts into a tar.gz file
+------------------------------------------
+
+The directory structure where you saved your PyTorch model should look something like the following:
+
+|   my_model
+|           |--model.pth
+|        
+|           code
+|               |--inference.py
+|               |--requirements.txt
+
+Where ``requirments.txt`` is an optional file that specifies dependencies on third-party libraries.
+
+With this file structure, run the following command to package your model as a ``tar.gz`` file:
+
+``tar -czf model.tar.gz my_model code``
+
+Upload model.tar.gz to S3
+-------------------------
+
+After you package your model into a ``tar.gz`` file, upload it to an S3 bucket by running the following python code:
+
+.. code:: python
+
+    import boto3
+    import sagemaker
+    s3 = boto3.client('s3')
+
+    from sagemaker import get_execution_role
+    role = get_execution_role()
+
+    response = s3.upload_file('model.tar.gz', 'my-bucket', '%s/%s' %('my-path', 'model.tar.gz'))
+
+Where ``my-bucket`` is the name of your S3 bucket, and ``my-path`` is the folder where you want to store the model.
+
+
+ You can also upload to S3 by using the AWS CLI:
+
+ .. code:: bash
+
+     aws s3 cp model.tar.gz s3://my-bucket/my-path/model.tar.gz
+
+
+To run this command, you'll need to have the AWS CLI tool installed. For information about installing the AWS CLI,
+see `Installing the AWS CLI <https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html>`_.
+
+Create a PyTorchModel object
+----------------------------
+
+Now call the :class:`sagemaker.pytorch.model.PyTorchModel` constructor to create a model object, and then call its ``deploy()`` method to deploy your model for inference.
+
+.. code:: python
+
+    pytorch_model = PyTorchModel(model_data='s3://my-bucket/my-path/model.tar.gz', role=role,
+                                 entry_point='inference.py')
+
+    predictor = pytorch_model.deploy(instance_type='ml.c4.xlarge', initial_instance_count=1)
+
+
+Now you can call the ``predict()`` method to get predictions from your deployed model.
+
+Attach an estimator to an existing training job
+===============================================
```

Comment:
This should probably be a level higher since it has nothing to do with deployments

```suggestion
***********************************************
Attach an estimator to an existing training job
***********************************************
```

alternatively, it could go in the training section